src/compat/ge_408.ml

```diff
@@ -0,0 +1,28 @@
+(**********************************************************************
```

Comment:
I suppose it could be helpful to have a single `compat.mli` to summarize both implementations, right?